sdk/daml-script/daml/Daml/Script/Internal/Questions/Crypto.daml

```diff
@@ -10,6 +10,7 @@ module Daml.Script.Internal.Questions.Crypto where
 
 #else
 
+-- | HIDE
```

Comment:
Rather than add a doc string (which didn't really add value), thought it best just to hide docs for this module